README.md

```diff
@@ -263,7 +263,8 @@ to configure a Quay.io application and deploy `image-controller`.
 
 The UI should now display the Lifecycle diagram for your application. In the Components
 tab you should be able to see your component listed and you'll be prompted to merge the
-automatically-created Pull Request (don't do that just yet).
+automatically-created Pull Request (don't do that just yet. we'll have it merged in
+section [Trigger the Release](#trigger-the-release)).
```

Comment:
Perhaps say "Trigger and Release section" rather then "section Trigger and Release"

Reply:
I prefer it the way it is.